### PR TITLE
resolves #1155 Dark mode - Utilisation de la librairie echart et refacto analytics

### LINKED ISSUE #1155: [TOCK STUDIO] - Dark mode- Use echart library
Use the echart library in the following screens so the the dark mode works correctly:

1. Analytics: replace the google library (incompatible with dark mode ) and refactoring of Activity and Behaviour screens

2. Model Quality: Use the echart library for the charts et replace the tables (angular material) by another component to adapt it to the dark mode

### PR DESCRIPTION
**Pull request by bakic**

### CODE REVIEW

File: bot/admin/server/src/main/kotlin/BotAdminVerticle.kt [hunk 1]
```diff
@@ -105,7 +105,7 @@ open class BotAdminVerticle : AdminVerticle() {
             }
         }
 
-        blockingJsonPost("/analytics/users", botUser) { context, query: UserSearchQuery ->
+        blockingJsonPost("/analytics/users1", botUser) { context, query: UserSearchQuery ->
```
Comment (vsct-jburet) [inline bot/admin/server/src/main/kotlin/BotAdminVerticle.kt:108]: Why users1 ?

Comment (bakic) [reply bot/admin/server/src/main/kotlin/BotAdminVerticle.kt]: @vsct-jburet This API is not used, I forgot to remove it.
now it's removed.

**Review by vsct-jburet — APPROVED** [review]

**Comment by francoisno** [comment]
Great work @bakic !
I have pushed several updates to the branch:

- branch is rebased
- your 2nd commit is squashed with the 1st
- I have added another commit with minor updates: missing mongo generated-sources + switching Users/Messages cards to have the same order between stats and graphs + renaming _Users_ to _Users (Dialogs)_*

(*) As far as I understand, those are not users but dialogs (`distinctBy { it.dialogId }`). Monitoring dialogs is important but it would be great too to provide stats correlated by `playerId`, monitoring actual user trafic - not only messages and dialogs. I propose:

- So far, we barely rename _Users_ to _Users (Dialogs)_ (as committed),
- Once actual user trafic is available, we will rename to _Users_ and _Dialogs_ respectively.

Is it OK for you @bakic ? If you approve this (and the additional commit) I can merge the request.

**Review by francoisno — APPROVED** [review]
Approving with an additional commit - requesting feedback from @bakic to merge.

**Comment by bakic** [comment]
Thanks @francoisno for this review.
I agree with you, I didn't want to go further in the development as I noticed that my commit is going far from a small refactoring,
I said that it may be a good start for further work on dialogs and users stats.
It's ok for me

**Pull request merged**